### PR TITLE
Fixes #21047: Move shared-files purge from crond to a task in relayd

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/21047

### CODE REVIEW

**Comment by amousset** [comment]
don't merge

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: relay/sources/relayd/tests/server.py [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
-            f.write("OK")
+            f.write('OK')
             f.close()
         else:
             self.send_error(404)
 
+
 server_address = ('', PORT)
 httpd = HTTPServer(server_address, PolicyServer)
-httpd.socket = ssl.wrap_socket(
```
Comment (amousset) [inline relay/sources/relayd/tests/server.py:46]: this method was deprecated since last year...
python keeps us entertained

**Comment by amousset** [comment]
PR updated with a new commit

**Review by VinceMacBuche — APPROVED** [review]

**Pull request merged**